recipe/sandbox/linear_algebra/vector_test.cc

```diff
@@ -0,0 +1,33 @@
+#include "recipe/sandbox/linear_algebra/vector.h"
+#include <gtest/gtest.h>
+
+namespace recipe {
+namespace sandbox {
+namespace linear_algebra {
+
+TEST(Vector, CopyConstructor) {
+  Vector v(3);
+  Vector vv(v);
+
+  EXPECT_EQ(v(0), vv(0));
```

Comment:
I should have define `bool operator==(const Vecotr& other) const` in `Vector` class.